src/components/15Years/15YearsGlobe.vue

```diff
@@ -104,13 +104,21 @@ export default {
 			const results = this.datastore.getNearest(world.lat, world.lon, 500, 1);
 			if (results) {
 				const result = results[0][0];
-				const offset = window.innerWidth < 681 ? -20 : 0;
+				const isMobileLayout = window.innerWidth < 681;
+				const offset = isMobileLayout ? -20 : 0;
 				this.gkview.animateToLatLon(result.lat + offset, result.lon);
 				const callout = new CalloutDefinition(result.lat, result.lon, PinCallout, result.properties);
 				callout.altitude = 0.035;
 				this.calloutManager.replaceCallouts([...this.callouts, callout]);
 				this.$emit('selectcountry', result.properties);
 				this.currentCallout = callout;
+
+				if (isMobileLayout) {
+					console.log(window.scrollY, window.innerHeight);
+					if (window.scrollY > window.innerHeight * 0.45) {
```

Comment:
@emuvente This check is a good rough estimate but should get us moving forward. We might later like to just add a check to see if the Country element is in the viewport.

Reply:
agreed